hack/spc_build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$SPC" != "true" ]]
+then
+    echo "This script is intended to be executed in an SPC,"
+    echo "by run_ci_tests.sh. Using it otherwise may result"
+    echo "in unplesent side-effects."
```

Comment:
unplesent to unpleasant

Reply:
ack :) will change in the next force push ... i just copied it from my other repos, I gotta change it there as well :\